### PR TITLE
Update _index.en.md

### PR DESCRIPTION
**Pull request by rburch**

### CODE REVIEW

File: docs/manual/core-extensions/news/_index.en.md [hunk 1]
```diff
@@ -1,17 +1,13 @@
 ---
 title: 'News/Blog extension'
-description: 'This allows news entries to be managed in the back end and output using front-end modules.'
+description: 'This allows news entries to be managed in the back end and output to the front-end using front-end modules.'
```
Comment (xchs) [inline docs/manual/core-extensions/news/_index.en.md]: Shouldn't we write "**back end**" and "**front end**" consistently without a hyphen?

File: docs/manual/core-extensions/news/_index.en.md [hunk 2]
```diff
@@ -1,17 +1,13 @@
...
 ---
 
-{{% notice warning %}}
-This article is machine translated.
-{{% /notice %}}
-
 **[contao/news bundle](https://packagist.org/packages/contao/news-bundle)**
 
-This allows news entries to be managed in the back end and output using front-end modules. A message can be structured as desired using content elements. Archive and commentary functions are also available.
+This allows news entries to be managed in the back end and output to the front-end using front-end modules. A news article can be structured the way you want using content elements. Archive and commenting functions are also available.
```
Comment (xchs) [inline docs/manual/core-extensions/news/_index.en.md]: Idem.

Comment (rburch) [reply docs/manual/core-extensions/news/_index.en.md]: Agreed. Thanks for pointing that out.

**Review by netzarbeiter — APPROVED** [review]
Thank you, Bob

**Pull request merged**